src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelThread.cs

```diff
@@ -30,6 +30,7 @@ public class KestrelThread
         private Queue<CloseHandle> _closeHandleRunning = new Queue<CloseHandle>();
         private object _workSync = new Object();
         private bool _stopImmediate = false;
+        private bool _initCompleted = false;
```

Comment:
I prefer [the terser route](https://github.com/aspnet/KestrelHttpServer/commit/3fb33119eec2550a9434e8761ad4c744bd528370), but when in Rome...